src/LogWrapper.ts

```diff
@@ -27,36 +27,26 @@ function isMessageNoise(messageOrObject: MsgType[]) {
 interface HookshotLogInfo extends winston.Logform.TransformableInfo {
     data: MsgType[];
 }
-export default class LogWrapper {
-
-    private static isConfigured: boolean;
+export class GlobalLogger {
```

Comment:
Is there such thing as a module-private constructor in TS/JS?  If nothing outside `LogWrapper.ts` needs to / should build a child logger, the visibility of `GlobalLogger`'s constructor could be restricted to just this file.  Of course there's no harm in keeping this exposed; was mostly just wondering if such scoping exists.

Reply:
We're using it for the test sadly :(